docs/chapter2-payload-image-format.rst

```diff
@@ -0,0 +1,659 @@
+                                              as seconds in POSIX time format.
+   arch              R           string       Type of the architecture for which this
+                                              image is intended:
+                                              'x86", "x86_64", "arm", "arm64",
+                                              "riscv", "riscv64"
+   type              R           string       Type of the image. Must be "flat_binary".
+   compression       O           string       Compression used to reduce image size:
+                                              "none" - no compression (default)
+                                              "lzma" - Lempel-Ziv-Markov chain-Algorithm,
+                                              "lz4" - Lempel-Ziv-4
+                                              If Platform Init should not decompress the
+                                              data when loading it, this must be set to
+                                              "none".
+   data-offset       R           u32          Offset of image data, measured from the end
+                                              of the FIT metadata, i.e.
+                                              fdt_totalsize(fit) bytes after the start of
+                                              the FIT, aligned to a 4-byte boundary.
+   data-size         R           u32          Size of image data in bytes. For compressed
+                                              images, this is the size of the compressed
+                                              data. The size of the uncompressed data is
+                                              stored in the 'uncomp-size' property.
+   load              R / O       u32 / u64    This should not normally be needed, since
+                                              Payloads should be loadable to any suitable
+                                              address.
+                                              Where that is not possible: This is the
+                                              required load address for the image. Value
+                                              Type matches the machine word size. This
+                                              must be provided for the 'firmware' image in
+                                              a configuration, but is optional for others.
+                                              Images without a load address can be loaded
+                                              to any suitable location. They can also be
+                                              left where they are in the FIT, i.e. not
+                                              loaded at all.
+   description       R           string       Textual description of the image. This may
+                                              be displayed to the user in some cases.
```

Comment:
description was already given in line 186.

Reply:
Done.